database/src/schema/permission_group.rs

```diff
@@ -104,6 +105,16 @@ impl PermissionGroup {
             .execute(executor)
             .await.map(|_| ())
     }
+
+    pub async fn delete<'e, E: sqlx::Executor<'e, Database = sqlx::Postgres>>(
```

Comment:
Unit test it, as above.

Reply:
This was the same argument as above, originally I was trying to gate it behind `#[cfg(test)]`, but unlike above, this function will be used eventually, so I will add tests for this (and the other one, and honestly, I'll just add it for Acocunt too at this point lol)

Reply:
As above, but I think if this is only supposed to be called in tests, then let's not unit test it, thats kinda ridiculous :smile:.